data-prepper-plugins/opensearch/build.gradle

```diff
@@ -81,8 +81,10 @@ configurations.all {
         force 'com.fasterxml.jackson.core:jackson-core:2.12.5'
         force 'com.fasterxml.jackson:jackson-bom:2.12.5'
         force 'com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:2.12.5'
+        force 'commons-codec:commons-codec:1.15'
         force 'commons-logging:commons-logging:1.2'
         force 'org.apache.httpcomponents:httpclient:4.5.13'
+        force 'org.apache.httpcomponents:httpcore:4.4.13'
         force "org.hdrhistogram:HdrHistogram:2.1.12"
         force 'joda-time:joda-time:2.10.10'
```

Comment:
not for this PR, but we really shouldn't be using Joda time since it is now included in the JDKs.

Reply:
This *might* be here because the OpenSearch build plugin brings in its own dependencies.

Reply:
Ok, we can dive into it later.